tests/microbenchmarks.py

```diff
@@ -2,7 +2,7 @@
 from os import path
 sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
 
-from utils import *
+from .utils import *
```

Comment:
At column 1: (F403) 'from .utils import *' used; unable to detect undefined names